### DISCUSSION TITLE
[New Feature] Add `SideBarSide` to allow map-makers to override the players sidebar theme.

### DISCUSSION THREAD

**Discussion by CCHyper** [Vinifera-Developers/Vinifera#1478] [Feature Requests]
### Description:
Tiberian Sun already has a map INI key called `SpeechSide` which allows the map-maker to override the player houses speech set. A new INI key to also allow the sidebar artwork to be overridden would be a good addition to this and add more control to map-makers.

### Possible Implementation:
Adding the INI key `SpeechSide` to ScenarioClass, which would be loaded from the scenario INI.

**Comment by MikillRosen** [comment]
Even better, make both of these INI flags applicable to rules.ini
